### PR TITLE
#24465 Updated min-height in _DevtoolsDialog.pcss

### PR DESCRIPTION
**Pull request by ABHIXIT2**
Most of the code for this window(Dev Tools Edit) is a same as the previous window(Dev Tools) which works for the Dev Tools window but not for this window.

Issue:
Here,the min-Height was set to 250px(without min-height edit window shrinks to Zero) which was lesser than the previous window's comperitive Height.

Solution:
I've had changed the height to 48.5vh(works according to the window size) 

https://user-images.githubusercontent.com/114552982/222381429-8966ea2c-9a3f-4832-97be-b6855afd52e9.mp4

After the changes :

https://user-images.githubusercontent.com/114552982/222434581-16e5f273-81ce-4742-9661-500b5ca4c449.mp4

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * #24465 Updated min-height in _DevtoolsDialog.pcss ([\#10264](https://github.com/matrix-org/matrix-react-sdk/pull/10264)). Contributed by @ABHIXIT2.

### CODE REVIEW

**Comment by ABHIXIT2** [comment]
Anyone please review and atleast inform me @kerryarchibald
@SimonBrandner

File: res/css/views/dialogs/_DevtoolsDialog.pcss [hunk 1]
```diff
@@ -79,7 +79,7 @@ limitations under the License.
 .mx_DevTools_textarea {
     font-size: $font-12px;
     max-width: 684px;
-    min-height: 250px;
+    min-height: 32em;
```
Comment (kerryarchibald) [inline res/css/views/dialogs/_DevtoolsDialog.pcss:82]: Where does this magic number come from? Can you add a comment to explain?

**Review by kerryarchibald — CHANGES_REQUESTED** [review]
Please include a description of the change and before/after screenshots in the PR description. (This is explained in the [contributing guide](https://github.com/vector-im/element-web/blob/develop/CONTRIBUTING.md#how-to-contribute))

I believe the issue 24465 refers the size of the dialog itself not expanding, rather than the textbox. You can see this difference by following the steps in the issue.

File: res/css/views/dialogs/_DevtoolsDialog.pcss (hunk 1 above)
Comment (ABHIXIT2) [reply res/css/views/dialogs/_DevtoolsDialog.pcss:82]: Most of the code for this window(Dev Tools Edit) is a same as the previous window(Dev Tools) which works for the Dev Tools window but not for this window.

Issue:
Here,the min-Height was set to 250px(without min-height edit window shrinks to Zero) which was lesser than the previous window's comperitive Height.

Solution:
I've had changed the height to 32 em which is working fine...Rn.

But I've found out another issue with the window so I am working on that too and I've mentioned ity in the pull request.

**Comment by ABHIXIT2** [comment]
I've discovered a new issue so please look into that and suggest me if we need to file another issue or just work on it.
@kerryarchibald @ndarilek

File: res/css/views/dialogs/_DevtoolsDialog.pcss (hunk 1 above)
Comment (ABHIXIT2) [reply res/css/views/dialogs/_DevtoolsDialog.pcss]: I've mad some changes according to requirement.

**Comment by ndarilek** [comment]
Why am I being pinged?

**Comment by ABHIXIT2** [comment]
> ```
>   Why am I being pinged?
> ```
Sorry sir, I did it by mistake 🙇

**Comment by ABHIXIT2** [comment]
@kerryarchibald  Please review!!

**Review by richvdh — CHANGES_REQUESTED** [review]
I don't understand what this PR is doing. Notably, it does not appear to fix #24465, at least on my monitor.

The PR description says what is changing (though it is somewhat confusing to read), but not what it is attempting to fix. Please say what you are trying to achieve instead of what you are changing (which I can see from the diff).

Likewise, please update the PR title to summarise what you are fixing, bearing in mind that this will end up in the *user-visible* changelog.

Finally, please sign-off your changes.

https://github.com/vector-im/element-web/blob/develop/CONTRIBUTING.md contains all this information: please read it.

**Pull request closed without merging**